tests_v2/test_workspace_create_background_starts_terminal.py

```
@@ -0,0 +1,83 @@
+                "background eager load should not switch the selected workspace",
+            )
+        finally:
+            try:
+                marker_path.unlink()
+            except FileNotFoundError:
+                pass
+            if created_workspace:
+                try:
+                    c.close_workspace(created_workspace)
+                except Exception:
+                    pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)